filing_history_descriptions.yml

```diff
@@ -910,12 +910,12 @@ description:
     'withdrawal-of-the-persons-with-significant-control-register-information-from-the-public-register' : "**Withdrawal** of the persons' with significant control register information from the public register"
     'withdrawal-of-the-members-register-information-from-the-public-register' : "**Withdrawal** of the members' register information from the public register"
-    'elect-to-keep-the-llp-members-register-information-on-the-public-register' : "**Elect** to keep the LLP members' register information on the public register"
-    'elect-to-keep-the-llp-members-residential-address-register-information-on-the-public-register' : "**Elect** to keep the LLP members' residential address register information on the public register"
-    'elect-to-keep-the-llp-persons-with-significant-control-register-information-on-the-public-register' : "**Elect** to keep the LLP persons' with significant control register information on the public register"
-    'withdrawal-of-the-llp-members-register-information-from-the-public-register' : "**Withdrawal** of the LLP members' register information from the public register"
-    'withdrawal-of-the-llp-members-residential-address-register-information-from-the-public-register' : "**Withdrawal** of the LLP members' residential address register information from the public register"
-    'withdrawal-of-the-llp-persons-with-significant-control-register-information-from-the-public-register' : "**Withdrawal** of the LLP persons' with significant control register information from the public register"
+    'elect-to-keep-the-limited-liability-partnership-members-register-information-on-the-public-register' : "**Elect** to keep the members' register information on the public register"
+    'elect-to-keep-the-limited-liability-partnership-members-residential-address-register-information-on-the-public-register' : "**Elect** to keep the members' residential address register information on the public register"
+    'elect-to-keep-the-limited-liability-partnership-persons-with-significant-control-register-information-on-the-public-register' : "**Elect** to keep the persons' with significant control register information on the public register"
+    'withdrawal-of-the-limited-liability-partnership-members-register-information-from-the-public-register' : "**Withdrawal** of the members' register information from the public register"
+    'withdrawal-of-the-limited-liability-partnership-members-residential-address-register-information-from-the-public-register' : "**Withdrawal** of the members' residential address register information from the public register"
+    'withdrawal-of-the-limited-liability-partnership-persons-with-significant-control-register-information-from-the-public-register' : "**Withdrawal** of the persons' with significant control register information from the public register"
```

Comment:
See note above, but if there are gong to be changes like this, end users will probably need to maintain enumerable files with both the new and the old values, or alternatively search through every reference to the enumerable value in their database.